app/usr/lib/systemd/system/pulp_resource_manager.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
No problem, I've run into that many times before!  It has the correct behavior but is just difficult to tell what happened from the diffs.